fs/sync.go

```diff
@@ -37,6 +37,10 @@ type syncCopyMove struct {
 }
 
 func newSyncCopyMove(fdst, fsrc Fs, Delete bool, DoMove bool) *syncCopyMove {
+
+	// Don't track renames for remotes without rename support.
+	_, canMove := fdst.(Mover)
```

Comment:
This should be Mover or Copier as some remotes (eg S3) don't have copy but do have Move - and you can synthesize Move with Copy+Delete